### ISSUE TITLE
npm install fails with ENOLOCAL on a11y

### ISSUE THREAD

**Issue by jvelo** [youknowriad/standalone-gutenberg#4]
When I try and run `npm install` on master, it fails with the following error :

```
$ npm install
npm ERR! code ENOLOCAL
npm ERR! Could not install from "node_modules/gutenberg/packages/a11y" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jerome/.npm/_logs/2018-10-25T15_44_59_073Z-debug.log
```

NPM version : 6.3.0
node : v8.11.3

**Comment by youknowriad** [comment]
Sorry this repo is not maintained any more and it's not going to be refreshed. Something like this might end up in the root Gutenberg repository in the future.

**Comment by jvelo** [comment]
@youknowriad thanks for the prompt answer. May I ask, is supporting a standalone mode an objective in the development of Gutenberg ?

**Comment by youknowriad** [comment]
Yes, it is and it's already possible, just not well documented (See how calypso is integrating it for instance)

**Comment by Grawl** [comment]
@jvelo hey I just fixed this problem 
https://github.com/Grawl/standalone-gutenberg/commit/7d4e3fb2e5ff809a5f6f1b4c7b49bae0d2e9b4e9

**Comment by Grawl** [comment]
@youknowriad do you want a PR? I want to update Gutenberg and fix here and there

**Comment by youknowriad** [comment]
I'd be ok with a PR. Ideally it would use the npm packages published by Gutenberg now and avoid referencing the Gutenberg package in package.json.

**Comment by Grawl** [comment]
I just install the same version that was installed by last commit date of this repo. Now it should work exactly like it works when you commit it.

**Comment by Grawl** [comment]
BTW image widget is broken even in [demo](http://gutenberg-standalone.surge.sh/)

**Comment by Grawl** [comment]
I just found this https://github.com/front/gutenberg-js
